src/commands/commander.h

```diff
@@ -182,9 +228,11 @@ struct CommandAttributes {
   // commander object generator
   CommanderFactory factory;
 
+  uint64_t InitialFlags() const { return flags_; }
```

Comment:
Minor question: why `InitialFlags()` in some cases?

Reply:
Some times we cannot get the cmd args, so maybe the initial flags is needed. Also I currently don't want to touch some legacy files.